src/components/SQForm/SQFormRadioButtonGroup.tsx

```diff
@@ -1,13 +1,37 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import Grid from '@material-ui/core/Grid';
-import RadioGroup from '@material-ui/core/RadioGroup';
-import FormHelperText from '@material-ui/core/FormHelperText';
-import FormControl from '@material-ui/core/FormControl';
-import FormLabel from '@material-ui/core/FormLabel';
+import {
+  Grid,
+  RadioGroup,
+  RadioProps,
+  FormHelperText,
+  FormControl,
+  FormLabel
+} from '@material-ui/core';
 import SQFormRadioButtonGroupItem from './SQFormRadioButtonGroupItem';
+import BaseFieldProps from 'types/BaseFieldProps';
 import {useForm} from './useForm';
 
+interface RadioButtonInputItemProps {
+  value: string | boolean | number;
+  label: string;
+  isDisabled?: boolean;
+  isRowDisplay?: boolean;
+  inputProps?: RadioProps;
+}
+
+interface SQFormRadioButtonGroupProps extends BaseFieldProps {
```

Comment:
I don't think we want this interface to extend `BaseFieldProps` since we don't use a label prop, I'm surprised TS wasn't complaining since `label` is not optional on `BaseFieldProps`